GitUI/Properties/Images.resx

```diff
@@ -799,4 +799,7 @@
   <data name="DocumentTree" type="System.Resources.ResXFileRef, System.Windows.Forms">
     <value>..\Resources\Icons\DocumentTree.png;System.Drawing.Bitmap, System.Drawing, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a</value>
   </data>
+  <data name="PullRequest" type="System.Resources.ResXFileRef, System.Windows.Forms">
+    <value>..\Resources\Icons\PullRequest.png;System.Drawing.Bitmap, System.Drawing, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a</value>
```

Comment:
I've just noticed we're adding a 3rd party icon, please have it recorded into `\gitextensions\GitUI\Resources\Icons\originals\_Icon Sources.txt` as well as copy the image into this folder.